x-pack/plugins/stack_connectors/server/connector_types/gemini/gemini.ts

```diff
@@ -395,9 +395,7 @@ const formatGeminiPayload = ({
       temperature,
       maxOutputTokens: DEFAULT_TOKEN_LIMIT,
     },
-    ...(systemInstruction
-      ? { system_instruction: { role: 'user', parts: [{ text: systemInstruction }] } }
```

Comment:
@pgayvallet Was there a reason for adding `role`? I don't see it in the [API docs](https://ai.google.dev/gemini-api/docs/system-instructions?lang=rest). I think all we need is `parts`